### PR TITLE
Adds class type to controller options

### PR DESCRIPTION
**Pull request by RipleyTom**
Lets you play with your guitar with mmjoystick on certain guitar hero(tested on 5).
Also lets use your pad as a guitar on the same games(might need to rebind buttons for usability though).

### CODE REVIEW

File: rpcs3/rpcs3qt/pad_settings_dialog.cpp [hunk 1]
```diff
@@ -169,6 +169,14 @@ pad_settings_dialog::pad_settings_dialog(QWidget *parent)
 	// Refresh Button
 	connect(ui->b_refresh, &QPushButton::clicked, this, &pad_settings_dialog::RefreshInputTypes);
 
+
+	ui->chooseClass->addItem("Standard(Pad)");
+	ui->chooseClass->addItem("Guitar");
+	ui->chooseClass->addItem("Drum");
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/pad_settings_dialog.cpp]: You should use tr() for these

Comment (Megamouse) [reply rpcs3/rpcs3qt/pad_settings_dialog.cpp]: Also no space in front of ( xD

**Comment by kamer1337** [comment]
i see you are working on controllers support. could you implement fake skateboard controller for tony hawks ride, tony hawks shred?

currently the games require skateboard  controller to play at all

**Comment by MSuih** [comment]
Tested this with Guitar Hero 5 and it works, controller is detected as guitar or drum and the buttons do work (although the game runs so badly it's hard to time anything). So in that sense this PR is ready to be merged.

The controller config screen needs rework though, with the current one you need trial and error to figure out where the buttons should be mapped to. But Megamouse can worry about that after this PR is merged.

**Comment by AniLeo** [comment]
On the side of the guitar you have the symbols for the buttons, map them appropriately and it works, when testing I managed to map correctly at first attempt.

**Comment by MSuih** [comment]
Oh right, should've mentioned but I used a normal controller to test this. People using xbox peripherals or perhaps keybords (a la Frets of Fire) would find it useful as well.

**Pull request merged**